source/simulator/assembly.cc

```diff
@@ -997,7 +997,7 @@ namespace aspect
                                       introspection.component_masks.velocities,
                                       constant_modes);
 
-    Mp_preconditioner.reset (new LinearAlgebra::PreconditionILU());
+    Mp_preconditioner.reset (new LinearAlgebra::PreconditionAMG());
```

Comment:
you are not setting any options here, is that on purpose? Are the defaults for AMG good?